### PR TITLE
mailing: foolproof message singular if one recipient

### PR DESCRIPTION
**Pull request by Melyodas**
Signed-off-by: Matthieu Moatti <moatti_m@epita.fr>

### CODE REVIEW

**Comment by seirl** [comment]
I don't think i18n works for "person"/"people"

**Comment by Melyodas** [comment]
> I don't think i18n works for "person"/"people"

What do you mean ?

File: prologin/mailing/forms.py [hunk 1]
```diff
@@ -60,15 +60,15 @@ class Meta:
...
+    FOOLPROOF_PHRASE = _("i am fine bothering %(n)s %(p)s")
 
     class Meta:
         model = mailing.models.Batch
         fields = ('name', 'template', 'query',)
 
     @classmethod
     def foolproof_field(cls, count):
-        phrase = cls.FOOLPROOF_PHRASE % {'n': count}
+        phrase = cls.FOOLPROOF_PHRASE % {'n': count, 'p' : 'people' if n > 1 else 'person'}
```
Comment (seirl) [inline prologin/mailing/forms.py:71]: No i18n.

**Review by seirl — CHANGES_REQUESTED** [review]

**Comment by Melyodas** [comment]
This is useless.

**Pull request closed without merging**